package.json

```diff
@@ -10,7 +10,7 @@
 	},
 	"scripts": {
 		"wmr": "yarn workspace wmr run",
-		"demo": "yarn workspace @examples/demo run",
+		"demo": "yarn workspace @examples/demo dev",
```

Comment:
Nit: This was probably included by accident into the changes. With the previous variant we can do stuff like `yarn demo dev` or `yarn demo build` to call the different package scripts.